include/appfwk/DAQModule.hpp

```diff
@@ -99,10 +156,34 @@ class DAQModule : public NamedObject
    *  Non-accepted commands or failure should return an ERS exception
    * indicating this result.
    */
-  virtual void execute_command(const std::string& cmd, const std::vector<std::string>& args) = 0;
+  void execute_command(const std::string& name, const std::vector<std::string>& args = {});
```

Comment:
I recommend that execute_commands stays virtual to allow optimisations or specific behaviour. 
If we are changing this interface we might as well change the argument to a json.

Reply:
This was done on purpose to avoid `DAQModule` bypass command registration.